src/Datadog.Trace.ClrProfiler.Managed/ExtensionMethods/LongExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.ClrProfiler.ExtensionMethods
+{
+    internal static class LongExtensions
+    {
+        public static Guid GetGuidFromNativePointer(this long nativePointer)
```

Comment:
I worry about abusing extension method syntax. `long` is a pretty common type and this method is only used in 1-2 places in the entire solution.

Reply:
Would you rather it be a simple static method?